lib/puppet/parameter/value_collection.rb

```diff
@@ -133,7 +133,9 @@ def newvalue(name, options = {}, &block)
     call_opt = options[:call]
     unless call_opt.nil?
       devfail "Cannot use obsolete :call value '#{call_opt}' for property '#{self.class.name}'" unless call_opt == :none || call_opt == :instead
-      Puppet.deprecation_warning(_("Property option :call is deprecated and no longer used. Please remove it."))
+      message = _("Property option :call is deprecated and no longer used.")
```

Comment:
`:call` probably shouldn't be translated here

Reply:
fixed